waiter/README.md

```diff
@@ -7,7 +7,7 @@ Waiter uses [Marathon](https://mesosphere.github.io/marathon/) to schedule servi
 Prerequisites:
 
 * Java 8
-* [Leiningen](http://leiningen.org/)
+* [Leiningen](http://leiningen.org/) (at least v2.8.1)
```

Comment:
Maybe we should just specify "v2.8.1", getting rid of the "at least" part. We've had issues with `lein` plugins (e.g., `voom`) breaking with new versions of Leiningen, so I'd rather not imply that newer versions are also supported.